org.eclipse.xtext/src/org/eclipse/xtext/formatting2/debug/TextRegionAccessToString.java

```diff
@@ -292,8 +304,19 @@ public boolean isHideIndentation() {
 		return hideIndentation;
 	}
 
+	/**
+	 * @deprecated use {@link #isHighlightOrigin()} instead
+	 */
+	@Deprecated
 	public boolean isHightlightOrigin() {
```

Comment:
as this is provisional api: i wonder if we need the deprecated. @szarnekow what do you think?

Reply:
I think we can remove the method from this provisional debug util.

Reply:
Removed the deprecated method